index.html

```diff
@@ -220,5 +231,45 @@ <h3 class="project-title">Multi-Post Stories</h3>
         <img class="project-image" src="images/home/Placeholder.svg" alt="" />
       </div>
     </section>
+    <article class="contact">
+      <h1>CONTACT US</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form class="form-container" action="https://formspree.io/f/xyyopqep" method="post">
+        <input type="text" name="Name" id="name" maxlength="30" placeholder="Text Input" />
+        <input type="email" name="Email" id="mail" placeholder="Jeremy@gmail.com" />
+        <textarea
+          name="user_message"
+          id="message"
+          maxlength="500"
+          placeholder="Enter text here"
+        ></textarea>
+        <button class="form-button" type="submit">See Project</button>
+      </form>
```

Comment:
- [ ] Kindly fix the form as it is not submitting.
<img width="615" alt="Screenshot 2022-03-22 at 11 57 58 PM" src="https://user-images.githubusercontent.com/70269507/159574961-7c0c265a-e27f-4c54-bf17-f06a21bdf81d.png">